src/wavm/WAVMModuleCache.cpp

```diff
@@ -0,0 +1,61 @@
+#include <wavm/WAVMWasmModule.h>
```

Comment:
This whole `WAVMModuleCache` class used to be the `WasmModuleCache` and have a lot more unnecessary complexity.